arcgis-ios-sdk-samples/Utility Network/Configure subnetwork trace/ConfigureSubnetworkTraceViewController.swift

```diff
@@ -0,0 +1,391 @@
+// Copyright 2020 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class ConfigureSubnetworkTraceViewController: UITableViewController {
+    // Reference to the switches.
+    @IBOutlet weak var barriersSwitch: UISwitch!
+    @IBOutlet weak var containersSwitch: UISwitch!
+    // References to interactable cells.
+    @IBOutlet weak var attributesCell: UITableViewCell!
+    @IBOutlet weak var comparisonCell: UITableViewCell!
+    @IBOutlet weak var valueCell: UITableViewCell!
+    @IBOutlet weak var addConditionLabel: UILabel!
+    @IBOutlet weak var resetLabel: UILabel!
+    @IBOutlet weak var traceLabel: UILabel!
+    // References to labels.
+    @IBOutlet weak var attributeLabel: UILabel!
+    @IBOutlet weak var comparisonLabel: UILabel!
+    @IBOutlet weak var valueLabel: UILabel!
+    // References to cells that act as buttons.
+    @IBOutlet weak var addConditionButton: UITableViewCell!
+    @IBOutlet weak var resetButton: UITableViewCell!
+    @IBOutlet weak var traceButton: UITableViewCell!
+    // Reference to the text view.
+    @IBOutlet weak var textView: UITextView!
+    // References to the switch actions.
+    @IBAction func barriersSwitchAction(_ sender: UISwitch) {
+        sourceTier?.traceConfiguration?.includeBarriers = sender.isOn
+    }
+    @IBAction func containersSwitchAction(_ sender: UISwitch) {
+        sourceTier?.traceConfiguration?.includeContainers = sender.isOn
+    }
```

Comment:
I'm not sure, but it is possible to get rid of these 2 actions, and directly access `barriersSwitch.isOn` property when it is needed.